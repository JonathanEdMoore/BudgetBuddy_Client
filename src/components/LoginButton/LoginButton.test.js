import React from 'react'
import ReactDOM from 'react-dom'
import LoginButton from './LoginButton'
import { BrowserRouter } from 'react-router-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('LoginButton smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><LoginButton /></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('LoginButton snapshot test', () => {
  it('renders the UI as expected', () => {
    const wrapper = shallow(<LoginButton />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})