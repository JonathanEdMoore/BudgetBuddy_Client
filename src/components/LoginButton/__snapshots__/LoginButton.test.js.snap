// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginButton snapshot test renders the UI as expected 1`] = `
<Link
  to="/login"
>
  Log In
</Link>
`;
